dragonchain/webserver/lib/transactions.py

```diff
@@ -133,9 +133,8 @@ def submit_transaction_v1(transaction: Dict[str, Any], callback_url: Optional[st
 
 def submit_bulk_transaction_v1(bulk_transaction: Sequence[Dict[str, Any]], api_key: "api_key_model.APIKeyModel") -> Dict[str, List[Any]]:
     """
-    Formats, validates and enqueues the transactions in
-    the payload of bulk_transaction
-    Returns dictionary of 2 lists, key "201" are successfully created txn ids and key "400" are transactions that failed to post (entire dto passed in from user)
+    Formats, validates and enqueues the transactions in the payload of bulk_transaction
+    Returns tuple of 2 lists, index 0 is successful txn ids and index 1 are transactions that failed to post (entire dto passed in from user)
```

Comment:
This returns a dictionary, not a tuple. Also seems unrelated...

Reply:
Not my change, must be from messed up rebase -- can do that again.

Reply:
This was somehow included from a rebase I apparently messed up.